compiler/rustc_session/src/options.rs

```diff
@@ -1832,7 +1832,7 @@ options! {
         specified passes to be enabled, overriding all other checks. In particular, this will \
         enable unsound (known-buggy and hence usually disabled) passes without further warning! \
         Passes that are not specified are enabled or disabled by other flags as usual."),
-    mir_include_spans: bool = (false, parse_bool, [UNTRACKED],
+    mir_include_spans: Option<bool> = (None, parse_opt_bool, [UNTRACKED],
```

Comment:
So wait, is this like... `None` = NLL dump only, `Some(true)` = always, `Some(false)` = never?

Reply:
If so, I'd prefer if we actually make this a ternary like `yes/no/nll` so people can choose all three. Having a default that is not possible to explicitly name is icky.

Reply:
Good point. I've made it a ternary now, and the "nll" default is considered "off" for everyone except when dumping NLL MIR.